neurobooth_os/iout/split_xdf.py

```diff
@@ -94,7 +92,8 @@ def parse_xdf(xdf_path: str, device_ids: Optional[List[str]] = None) -> List[Dev
             continue
 
         device_id = device_data["info"]["desc"][0]["device_id"][0]
-        sensors_ids = json.loads(device_data["info"]["desc"][0]["sensor_ids"][0])  # Deserialize into list
+        sensors_ids = device_data["info"]["desc"][0]["sensor_ids"][0]
```

Comment:
Nit: might be better to not reuse the variable sensors_ids for both the original form and the json form (next line). Also, might be better to use "sensor_ids" rather than "sensors_ids".

Reply:
Thanks! Addressed this.